spec/features/quotes/index_spec.rb

```diff
@@ -0,0 +1,42 @@
+RSpec.describe 'quotes index' do
+  
+  it 'displays the top 10 quotes for a search' do
+    visit root_path
+
+    fill_in :keyword, with: "divided house"
+
+    click_on "Search for Quotes"
+
+    expect(current_path).to eq("/quotes")
+    expect(page).to have_content("Author:", count: 10)
+    expect(page).to have_content("Quote:", count: 10)
+    expect(page).to have_content("Categories:", count: 10)
+  end
+
```

Comment:
👍 Nice, I like this. 

You could also get all the quotes and test those in this test by doing something like: 

```ruby
quotes = Facade.method(query_word)
quotes.each do |quote|
 expect(page).to have_content (quote.content)
  expect(page).to have_content (quote.author)
# ... and same thing for tags